### PR TITLE
Sphinx extension remove unused code

### PR DESCRIPTION
**Pull request by sethfischer**
Remove unused code from Sphinx extension.

### CODE REVIEW

**Comment by jmwright** [comment]
@sethfischer Is this ready for review?

**Comment by sethfischer** [comment]
@jmwright yes, ready for review.

File: cadquery/cq_directive.py [hunk 1]
```diff
@@ -278,7 +269,7 @@ class cq_directive_vtk(Directive):
 
     has_content = True
     required_arguments = 0
-    optional_arguments = 2
+    optional_arguments = 0
```
Comment (adam-urbanczyk) [inline cadquery/cq_directive.py:272]: Could you explain (I'm a bit rusty on this code)?

**Review by adam-urbanczyk — APPROVED** [review]

File: cadquery/cq_directive.py (hunk 1 above)
Comment (sethfischer) [reply cadquery/cq_directive.py:272]: The directive `cadquery` takes no arguments so it is used as follows:

```rst
.. cadquery::

    import cadquery as cq
```

In the event that a directive accepts arguments it looks something like:

```rst
.. my-directive:: arg1 arg2

    content goes here.
```

Reference:

https://docutils.sourceforge.io/docs/howto/rst-directives.html

> - `required_arguments`: The number of required arguments (default: 0).
> - `optional_arguments`: The number of optional arguments (default: 0).

**Comment by jmwright** [comment]
I'll wait to review this until the conversation is resolved and all steps are green.

File: cadquery/cq_directive.py (hunk 1 above)
Comment (adam-urbanczyk) [reply cadquery/cq_directive.py:272]: Thanks, clear

**Comment by sethfischer** [comment]
The tests are failing with the following error which appears unrelated to this PR.

```
ImportError while importing test module '/home/appveyor/projects/cadquery-o18bh/tests/test_assembly.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../../miniconda/envs/cadquery/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/__init__.py:1: in <module>
    from cadquery import *
cadquery/__init__.py:10: in <module>
    from .occ_impl.geom import Plane, BoundBox, Vector, Matrix, Location
cadquery/occ_impl/geom.py:5: in <module>
    from OCP.gp import (
E   ImportError: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /home/appveyor/miniconda/envs/cadquery/lib/python3.10/site-packages/OCP.cpython-310-x86_64-linux-gnu.so)
_________________________
```

https://ci.appveyor.com/project/jmwright/cadquery-o18bh/builds/47288568/job/c68qv173he5q2hw8#L677

**Comment by jmwright** [comment]
We're getting this error on Linux and type errors on the other OSes as well across multiple PRs. @adam-urbanczyk do these errors have something to do with OCP 7.7.1.0 being available in conda-forge now? Do we need to merge #1343 first to fix these errors?

**Comment by jmwright** [comment]
@adam-urbanczyk I think that AppVeyor still uses Ubuntu 18.04 by default, whereas OCP is built on Ubuntu 20.04. That may be the cause of the libc version mismatch error in Linux.

https://www.appveyor.com/docs/build-environment/#build-worker-images

**Comment by adam-urbanczyk** [comment]
Yes, I'm aware. I'm trying different things (setting labels in the channel) to solve this issue.

**Review by jmwright — APPROVED** [review]
Looks good, thanks @sethfischer 

@adam-urbanczyk Shall I merge, or are there more changes that need to be made?

**Comment by adam-urbanczyk** [comment]
Done, thanks @sethfischer !

**Pull request merged**